src/components/Breadcrumbs/Breadcrumb.js

```diff
@@ -1,37 +1,24 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { Link } from "gatsby";
-import { handlePath } from "../../helpers/handlers";
+import { Breadcrumb } from "gatsby-plugin-breadcrumb";
 
 import "./Breadcrumb.css";
 
-const Breadcrumb = ({ crumbs }) => {
+const BreadcrumbComponent = ({ crumbs }) => {
```

Comment:
Please rename the file or change the className. Let's keep the filename, the component name, and the className same, they shouldn't differ.